snap/README.md

```diff
@@ -13,8 +13,8 @@ This directory contains the resources needed to build the Kata Containers
 
 ## Initial setup
 
-*Ubuntu 18.04*
-
+Kata Containers can be installed in any Linux distribution that supports
+[snapd](https://docs.snapcraft.io/installing-snapd).
 ```sh
 $ sudo apt-get install -y snapd snapcraft
```

Comment:
but here we are using `apt-get`, I think you should mention that for example the following command is to install snapd in ubuntu

Reply:
@devimc : Fixed.